### DISCUSSION TITLE
Simple_GP_Regression

### DISCUSSION THREAD

**Discussion by ssaxenaANL** [cornellius-gp/gpytorch#1663] [Q&A]
I am trying to run the simple GP regression notebook, however the training step is breaking the kernel without giving an error:
its breaking at loss calculation step:
loss = -mll(output, train_y)

![image](https://user-images.githubusercontent.com/61334206/122581537-c2281300-d01c-11eb-8df9-f0286c62d54f.png)

**Comment by gpleiss** [comment]
Can you please provide a stack trace? What is the error that you are seeing?
